### ISSUE TITLE
Error playing video with special characters.

### ISSUE THREAD

**Issue by luhanmcs** [cguZZman/plugin.googledrive#261]
_"ascii' codec can't encode characters in position 46-47: ordinal not in range"_
Error playing video with special characters.
Error happens on windows and android.
Kodi 18.9 works fine.
[https://paste.kodi.tv/oyusanuqir](url)
![screenshot00001](https://user-images.githubusercontent.com/22850716/128437569-a9b89315-c1bb-411b-985d-1f6edc3b61ce.png)

**Comment by RedSquirrel87** [comment]
I get the same error, for example with every "pokémon" movie (because of the "é" char). I didn't think it was such a long-date bug (I see it was reported in August), any news about it or how to fix this?

**Comment by RedSquirrel87** [comment]
Ok, I just analyzed the source code, to solve this issue just replace the following string at "resources/lib/addon.py"
`response_params = dict(urllib.parse.parse_qsl(response_text))`
with
`response_params = dict(urllib.parse.parse_qsl(response_text.decode('ASCII')))`
and then it will work

**Comment by andreus1610** [comment]
Yes It happened to me too! This happens when you try to open a file that contains an accent and in the configurations you have set a stream format other than Original.

Your change on the addon.py file appears to work.

Thanks!


> Ok, I just analyzed the source code, to solve this issue just replace the following string at "resources/lib/addon.py" `response_params = dict(urllib.parse.parse_qsl(response_text))` with `response_params = dict(urllib.parse.parse_qsl(response_text.decode('ASCII')))` and then it will work

**Comment by DVergil** [comment]
hello how can i do this?

**Comment by DVergil** [comment]
I use kodi on Android TV

**Comment by hungdry** [comment]
> Ok, I just analyzed the source code, to solve this issue just replace the following string at "resources/lib/addon.py" `response_params = dict(urllib.parse.parse_qsl(response_text))` with `response_params = dict(urllib.parse.parse_qsl(response_text.decode('ASCII')))` and then it will work

It works for me. Thanks

**Comment by hungdry** [comment]
> I use kodi on Android TV

Download code as zip file
Edit the line that RedSquirrel87 mentioned and zip again
Send that modified zip file to your box/android TV
Kodi -> setting -> add-on -> install from zip file
